cirq-google/cirq_google/calibration/workflow.py

```diff
@@ -243,7 +243,7 @@ def prepare_characterization_for_moments(
     operations and operations supported by gates_translator.
 
     Args:
-        circuit: Circuit to characterize.
+        circuit: Circuit or list of circuits to characterize.
```

Comment:
optional: consider renaming to `circuits`

Reply:
I also like it better but now it's a breaking change